application/controllers/teams_controller.py

```diff
@@ -4,21 +4,22 @@
 from flask import current_app
 from flask_socketio import join_room, leave_room
 from webargs.flaskparser import use_args
+from flask_jwt_extended import jwt_required, get_jwt_identity
 
 from .application_controller import ApplicationController
 from ..extensions import socketio
-from ..models.team import Team
+from ..models import Team, User
 from ..responders import respond_with
 from ..validators import teams_validator
 
-@socketio.on("JOIN_TEAM")
-def on_team_join(data):
+@socketio.on("ENTER_TEAM_SPACE")
+def on_enter_team_space(data):
```

Comment:
This will break presentation mode again as long as it isn't fixed on the front end as well I think?
I thought Luise didn't push this but it's probably faster to change it in the front end as well than removing it here again.